src/analyses/basePriv.ml

```diff
@@ -1802,7 +1802,7 @@ struct
   let write_global ?invariant ask getg sideg st x v = time "write_global" (Priv.write_global ?invariant ask getg sideg st x) v
   let lock ask getg cpa m = time "lock" (Priv.lock ask getg cpa) m
   let unlock ask getg sideg st m = time "unlock" (Priv.unlock ask getg sideg st) m
-  let sync reason ctx = time "sync" (Priv.sync reason) ctx
+  let sync reason man = time "sync" (Priv.sync reason) man
```

Comment:
Now looking at it, this seems wrong to begin with: all privatization functions only get a few things from `man` as separate arguments:
```suggestion
  let sync ask getg sideg st reason = time "sync" (Priv.sync ask getg sideg st) reason
```
The type was the same, but the timing was measured from partial application, so it would've always been close to 0 I think.